<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>785</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KnowNow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mulTab">
     <attribute name="title">
      <string>选择题</string>
     </attribute>
     <widget class="QGroupBox" name="choiceGroupBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>941</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>选择题</string>
      </property>
      <widget class="QCheckBox" name="AcheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>701</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="BcheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>711</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>B</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="CcheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>721</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="DcheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>741</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="EcheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>300</y>
         <width>721</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>E</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="FcheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>340</y>
         <width>731</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>F</string>
       </property>
      </widget>
      <widget class="QLabel" name="quesLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>871</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>国际航标协会（IALA）是一个非政府的、非获利的技术团体，成立于1957年，为来自于世界各地区的航标主管机关、生产厂商和咨询机构提供一个框架，共同致力于：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="imgLabel">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>90</y>
         <width>331</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="choiceNextButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>410</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>下一题</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>641</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>上一题回答情况</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>你的回答：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>正确答案：</string>
       </property>
      </widget>
      <widget class="QLabel" name="isTrueLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>72</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="answerLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>72</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>问题：</string>
       </property>
      </widget>
      <widget class="QLabel" name="questionLable">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>521</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>选项：</string>
       </property>
      </widget>
      <widget class="QLabel" name="optionsLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>551</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="binTab">
     <attribute name="title">
      <string>判断题</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>941</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>判断题</string>
      </property>
      <widget class="QLabel" name="checkLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>751</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>判断题国际航标协会（IALA）是一个非政府的、非获利的技术团体，成立于1957年，为来自于世界各地区的航标主管机关、生产厂商和咨询机构提供一个框架，共同致力于：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="trueRadioButton">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>300</y>
         <width>115</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>正确</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="falseRadioButton">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>300</y>
         <width>115</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>错误</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="checkNextButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>410</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>下一题</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>641</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>上一题回答情况</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>72</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>你的回答：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>正确答案：</string>
       </property>
      </widget>
      <widget class="QLabel" name="isTrueLabel_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="answerLabel_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>72</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>问题：</string>
       </property>
      </widget>
      <widget class="QLabel" name="checkQuestionLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>521</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>切换主题</string>
    </property>
    <addaction name="actionlight"/>
    <addaction name="actiondark_theme"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlight">
   <property name="text">
    <string>light theme</string>
   </property>
  </action>
  <action name="actiondark_theme">
   <property name="text">
    <string>dark theme</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
